account_tax_balance/README.rst

```diff
@@ -0,0 +1,63 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+============
+Tax Balances
+============
+
+This module allows to compute tax balances within a certain date range.
+It depends on date_range module and exposes 'compute' methods that can be called by other modules (like localization ones)
+
+Usage
+=====
+
+Accounting --> Reporting --> Taxes Balance
+
+Select the company, the date range, the target moves and 'open taxes'
+
+.. figure:: /account_tax_balance/static/description/tax_balance.png
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/91/11.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/account-financial-reporting/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed feedback.
+
+Credits
+=======
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
+* Lorenzo Battistini <lorenzo.battistini@agilebg.com>
+* Giovanni Capalbo <giovanni@therp.nl>
+* Tecnativa - Antonio Espinosa
+* Tecnativa - Pedro M. Baeza
```

Comment:
Standard format is "Lorenzo Battistini <lorenzo.battistini@agilebg.com>"

Isn't it?

Reply:
I prefer not to put here email